content-src/reducers/reducers.js

```diff
@@ -21,8 +21,34 @@ function setRowsOrError(type) {
   };
 }
 
+function setSearchState(type) {
```

Comment:
@k88hudson i thought it might be useful to keep the reducer here now, for future search reducer stuff?